aggsender/flows/flow_pp.go

```diff
@@ -182,3 +182,8 @@ func (p *PPFlow) ValidateCertificate(ctx context.Context, cert *agglayertypes.Ce
 
 	return nil
 }
+
+// Signer returns the signer used to sign the certificate
+func (p *PPFlow) Signer() signertypes.Signer {
+	return p.signer
```

Comment:
Rename it to certificateSigner (to be the same as in the `aggchain prover` flow)?

Reply:
Fix: https://github.com/agglayer/aggkit/pull/951/commits/1e568446ce41e0d222f00c93cd370a7eb02901a0